bitchat/ViewModels/ChatViewModel.swift

```diff
@@ -2977,24 +2977,21 @@ final class ChatViewModel: ObservableObject, BitchatDelegate {
         // No-op; avoid forcing synchronize on resign
     }
     
-    @objc func applicationWillTerminate() {
-        // Send leave message to all peers
-        meshService.stopServices()
-        
+    /// Save identity state without stopping services (for backgrounding)
+    func saveIdentityState() {
         // Force save any pending identity changes (verifications, favorites, etc)
         identityManager.forceSave()
```

Comment:
Why not call this when the identity is actually verified? Because it's a user-initiated process of scanning the QR adding a small delay until all the verification-related process is completed would be an acceptable delay